api/app/controllers/api/v1/statistics_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Api
+  module V1
+    # Statistics controller
+    class StatisticsController < ApplicationController
+
+      resourceful! Statistics, authorize_options: { except: [:index] } do
```

Comment:
We actually do need to authorize the index route. These statistics should only be accessible to admin users, not the general public.